src/components/Table.stories.tsx

```diff
@@ -1,101 +1,93 @@
-import 'braid-design-system/reset';
 import 'loki/configure-react';
 
 import { Text } from 'braid-design-system';
-import React, { type ComponentProps, Fragment } from 'react';
-
-import {
-  type BraidArgs,
-  defaultArgTypes,
-  defaultArgs,
-} from '../storybook/controls';
-import { BraidStorybookProvider, withRouter } from '../storybook/decorators';
+import React, { Fragment } from 'react';
+import type { Meta, StoryObj } from 'sku/@storybook/react';
 
 import { Table as Component } from './Table';
 import { TableRow } from './TableRow';
 export default {
+  title: 'Standalone/Table',
+  component: Component,
   args: {
-    braidThemeName: defaultArgs.braidThemeName,
-    size: defaultArgs.size,
-    width: 'undefined',
+    size: 'standard',
+    width: undefined,
   },
   argTypes: {
-    braidThemeName: defaultArgTypes.braidThemeName,
-    size: defaultArgTypes.size,
+    size: {
+      control: { type: 'radio' },
+      options: ['standard', 'large'],
+    },
     width: {
       control: { type: 'radio' },
       mapping: { undefined, full: 'full' },
-      options: ['undefined', 'full'],
+      options: [undefined, 'full'],
```

Comment:
If we're allowed a literal `undefined` in options now, does that mean we don't need `mapping` anymore?

Reply:
Ah good catch, nope we don't